libs/sdk-bindings/src/uniffi_binding.rs

```diff
@@ -277,8 +277,11 @@ impl BlockingBreezServices {
         .map_err(|e| e.into())
     }
 
-    pub fn fetch_reverse_swap_fees(&self) -> SdkResult<ReverseSwapPairInfo> {
-        rt().block_on(self.breez_services.fetch_reverse_swap_fees())
+    pub fn fetch_reverse_swap_fees(
+        &self,
+        send_amount_sat: Option<u64>,
```

Comment:
Let's take the opportunity to change to one struct argument? `ReverseSwapFeesRequest` ?

Reply:
Good idea, done.